common/lib/xmodule/xmodule/js/spec/video/video_player_spec.js

```diff
@@ -1011,6 +1011,21 @@ function(VideoPlayer, HLS) {
             });
         });
 
+        describe('Video duration', function() {
+            beforeEach(function() {
+                state = jasmine.initializePlayer();
+                spyOn(state.videoPlayer, 'duration').and.returnValue(61);
+            });
+
+            it('overrides the duration if not set', function(done) {
```

Comment:
@muhammad-ammar what's the purpose of `done` here?

Reply:
`done` is a callback which a test should call when the test is completed. [jasmine docs](https://jasmine.github.io/2.0/introduction.html#section-Asynchronous_Support)